main.js

```diff
@@ -1,81 +1,168 @@
 // Match ANSI escape code
-const pattern = /\[\d+;?\d*m/g;
-
-// ANSI escape code
-const colorCodes = {
-    '[2m': 'opacity:0.75',
-    '[31m': 'color:red',
-    '[32m': 'color:green',
-    '[33m': 'color:gold',
-    '[35m': 'color:purple',
-    '[36m': 'color:darkblue',
-    '[94m': 'color:lightblue',
-    '[0;39m': '</span>',
-    '[39m': '</span>',
+
+const escape_codes = new Map(
+    [
+        ['0', 'font-weight:normal'],
+        ['1', 'font-weight:bold'],
+        ['2', 'opacity:0.75'],
+        ['4', 'text-decoration:underline'],
+        ['30', 'color:darkslategray'],
+        ['31', 'color:darkred'],
+        ['32', 'color:darkgreen'],
+        ['33', 'color:darkgoldenrod'],
+        ['34', 'color:darkblue'],
+        ['35', 'color:indigo'],
+        ['36', 'color:darkcyan'],
+        ['39', 'color:lightgrey'],
```

Comment:
I think lightgrey is not dark enough, this is how it renders:
![image](https://user-images.githubusercontent.com/2212267/159125383-84e129db-bf24-481c-9ba1-64e49190952d.png)

With black:
![image](https://user-images.githubusercontent.com/2212267/159125433-14181731-a582-4c03-b099-330b68e1aa4d.png)